inc/type.class.php

```diff
@@ -1928,6 +1928,15 @@ static function install(Migration $migration) {
 
       $notepad = new Notepad();
       foreach ($allGenericObjectTypes as $genericObjectType => $genericObjectData) {
+         $itemtype = $genericObjectData['itemtype'];
+         if (! class_exists($itemtype, true)) {
+            // TRANS:  %1$s is itemtype name
+            $warning  = sprintf(__('Unable to load the class %1$s.', 'genericobject'), $itemtype);
+            // TRANS:  %1$s is itemtype name
+            $warning .= sprintf(__('You probably have garbage data in your database for this plugin but missing files in %1$s', 'genericobject'), GENERICOBJECT_DOC_DIR);
```

Comment:
Actually I think the bug occurs when there are itemtypes defined in tables of the plugin, but at least one related class file is missing.  Then I did not used "or" in the sentence.